packages/dds/sequence/src/test/intervalCollection.fuzzUtils.ts

```diff
@@ -203,17 +208,32 @@ export function makeReducer(
 		removeRange: async ({ channel }, { start, end }) => {
 			channel.removeRange(start, end);
 		},
-		addInterval: async ({ channel }, { start, end, collectionName, id }) => {
+		addInterval: async (
+			{ channel },
+			{ start, end, collectionName, id, startSide, endSide },
+		) => {
 			const collection = channel.getIntervalCollection(collectionName);
-			collection.add(start, end, IntervalType.SlideOnRemove, { intervalId: id });
+			collection.add(
+				{ pos: start, side: startSide },
```

Comment:
might make some of this code a little less verbose if we generally work in terms of `SequencePlace` rather than `number | undefined` when generating operations. not a big deal though.